src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/main/Drawer.java

```diff
@@ -13,7 +13,7 @@
     /**
      * The Drawer draws itself onto the GraphicsContext at its position.
      * @param context The context that the object draws itself onto
-     * @param time The current time in nanoseconds as supplied by ApplicationTimer
+     * @param timeDiff The difference in time in minutes since the last call of draw.
```

Comment:
Ja.
Der Parameter wird im ApplicationTimer berechnet. Dabei wird der aktuell uebergebene Wert in Ns (den der ApplicationTimer bekomme) minus dem letzten Wert gerechnet. Dann / (1000000000 * 60) (Anzahl Ns pro Minute).
In Minuten wird es umgerechnet, um dann Drehzahlen die ja alle in U/ min gegeben sind leichter darstellen zu koennen und das nicht in jeder Klasse separat machen zu muessen.